kura/org.eclipse.kura.ai.triton.server/META-INF/MANIFEST.MF

```diff
@@ -25,18 +25,18 @@ Import-Package: com.google.common.base;version="25.0.0",
  javax.annotation;version="1.2.0",
  org.apache.commons.io;version="2.4.0",
  org.apache.commons.io.filefilter;version="2.11.0",
- org.bouncycastle.jcajce;version="1.68.0",
- org.bouncycastle.jcajce.interfaces;version="1.68.0",
- org.bouncycastle.jcajce.io;version="1.68.0",
- org.bouncycastle.jcajce.spec;version="1.68.0",
- org.bouncycastle.jcajce.util;version="1.68.0",
- org.bouncycastle.jce.provider;version="1.68.0",
+ org.bouncycastle.jcajce;version="1.77.0",
+ org.bouncycastle.jcajce.interfaces;version="1.77.0",
+ org.bouncycastle.jcajce.io;version="1.77.0",
+ org.bouncycastle.jcajce.spec;version="1.77.0",
+ org.bouncycastle.jcajce.util;version="1.77.0",
+ org.bouncycastle.jce.provider;version="1.77.0",
  org.bouncycastle.openpgp;version="1.71.0",
  org.bouncycastle.openpgp.jcajce;version="1.71.0",
  org.bouncycastle.openpgp.operator.jcajce;version="1.71.0",
- org.bouncycastle.util;version="1.68.0",
```

Comment:
Do we want to update also the following versions for consistency?

```
 org.bouncycastle.openpgp;version="1.71.0",
 org.bouncycastle.openpgp.jcajce;version="1.71.0",
 org.bouncycastle.openpgp.operator.jcajce;version="1.71.0",
```

Reply:
Oh! I have missed that!
I'll do it now

Reply:
Done